packages/types-algosdk/index.d.ts

```diff
@@ -200,15 +200,22 @@ export interface MultiSigAccount {
   addrs: string[];
 }
 
+// Stateful Smart Contract Schema
 export interface SSCStateSchema {
-  key: string;
+  key: string | Uint8Array;
   value: {
     type: number;
-    bytes: string;
+    bytes: string | Uint8Array;
```

Comment:
why do we have `string` type for bytes? Can we remove `string`? Otherwise, can we add a ne `value` filed (string: string)?

Reply:
SDk returns base64 encoded data for key value pairs represent as bytes. here is the chat link: [https://discord.com/channels/754286650540032030/754286650540032033/788166652642328596]() 

Adding a new value to this type would lead in an incorrect structure.

Reply:
so, if we store here `base64` string, then why do we need `Uint8Array`?

Reply:
Because we are representing bytes as `Uint8Array`. Could you check: [https://discord.com/channels/754286650540032030/754286650540032033/788171546651918376]()

Reply:
Yes, that why in my first comment I was hoping to use only `Uint8Array`, not a string.  It's still not clear to me why we can't operate only on bytes and use base64 only for IO.